src/index.js

```diff
@@ -567,15 +574,15 @@ function createConfig(options, entry, format, writeMeta) {
 							};
 						},
 					},
-					buble({
-						exclude: 'node_modules/**',
-						jsx: options.jsx || 'h',
-						objectAssign: options.assign || 'Object.assign',
-						transforms: {
-							dangerousForOf: true,
-							dangerousTaggedTemplateString: true,
-						},
-					}),
+					// buble({
+					// 	exclude: 'node_modules/**',
+					// 	jsx: options.jsx || 'h',
+					// 	objectAssign: options.assign || 'Object.assign',
+					// 	transforms: {
+					// 		dangerousForOf: true,
```

Comment:
is there a way to catch this transform? Something like: https://www.npmjs.com/package/@babel/plugin-transform-for-of

Reply:
Like i said above probably a lot of e2e-tests or snippets which we can test.

Reply:
Sadly babel does not have this option and pretty sure it's not needed.